io.github.ad_on_is.Resticity.yml

```diff
@@ -0,0 +1,76 @@
+id: io.github.ad_on_is.Resticity
+runtime: org.gnome.Platform
+runtime-version: '3.38'
```

Comment:
this runtime has been EOL for years now. Use 46.

Reply:
Well, it's the only Platform that builds successfully with webkit2-4.0 (which is used by wails)

Reply:
no. it is EOL and has been for a couple of years.

Either the code should be updated to use the more recent WebKitGtk or you should build it.

Reply:
> no. it is EOL and has been for a couple of years.
> 
> Either the code should be updated to use the more recent WebKitGtk or you should build it.

Last time I tried building webkit, you explicitely told me not to do so, since it's already there.

Reply:
No.

I remember my word where "this is in the runtime". 

A proper answer could have been "it's the wrong version", instead you went on checking in binaries. 

I could waste my volunteer valuable time checking any random framework for language I don't, or I could rely on the knowledge of people submitting. I chose the latter.

Reply:
Look, I'm building this software in my free time too, and have never dealt with distributing apps for operating systems, like Flathub or alike.

I usually build web and server-side apps, and distribute them via Docker.

This is all new to me, and considering the sheer amount of fragmentation, like Flathub, AppImages, RPMs, etc... feels exhausting.

Also, the fact that the 3.38 runtime is still available made me think it's the right one for GTK2/3 apps, whereas 45+ is for GTK4, and for a moment I was excited to have a finally working build 😅

So, if you feel I'm wasting your time, I'm sorry.